nc_time_axis/tests/integration/test_plot.py

```diff
@@ -0,0 +1,41 @@
+"""Integration test for plotting data with non-gregorian calendar."""
+
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+import unittest
+
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import netcdftime
+import numpy as np
+
+import nc_time_axis
+
+
+class Test(unittest.TestCase):
+    def setUp(self):
+        # Make sure we have no unclosed plots from previous tests before
+        # generating this one.
+        plt.close('all')
+
+    def tearDown(self):
+        # If a plotting test bombs out it can leave the current figure
+        # in an odd state, so we make sure it's been disposed of.
+        plt.close('all')
+
+    def test_360_day_calendar(self):
+        datetimes = [netcdftime.datetime(1986, month, 30)
+                     for month in range(1, 6)]
+        cal_datetimes = [nc_time_axis.CalendarDateTime(dt, '360_day')
+                         for dt in datetimes]
+        line1, = plt.plot(cal_datetimes)
+        result_ydata = line1.get_ydata()
+        result_datetimes = [cdt.datetime for cdt in result_ydata]
+        np.testing.assert_array_equal(result_datetimes, datetimes)
+        self.assertIs('360_day', result_ydata[0].calendar)
```

Comment:
I suppose I could add a `__eq__` method to CalendarDateTime to save from having to always test the equality of the datetimes, and the calendars separately (which I have also had to do in a [test that was included in PR#12](https://github.com/lbdreyer/nc-time-axis/blob/1ff3e091c34052f0187756d64cd420af58cb677d/nc_time_axis/tests/unit/test_NetCDFTimeConverter.py#L21)).